avro/src/test/java/com/fasterxml/jackson/dataformat/avro/schema/EnumSchema422Test.java

```diff
@@ -0,0 +1,60 @@
+package com.fasterxml.jackson.dataformat.avro.schema;
+
+import org.junit.Test;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import com.fasterxml.jackson.dataformat.avro.AvroMapper;
+import com.fasterxml.jackson.dataformat.avro.AvroTestBase;
+
+// For [dataformats-binary#422]
+public class EnumSchema422Test extends AvroTestBase
+{
+    enum EnumType422 {
+        CARD_S("CARD-S");
+
+        private final String value;
+
+        EnumType422(String value) {
+            this.value = value;
+        }
+
+        @JsonValue
+        public String value() {
+            return this.value;
+        }
+    }
+
+    static class Wrapper422 {
+        public EnumType422 contract;
+    }
+
+    private final AvroMapper MAPPER = newMapper();
+
+    // For [dataformats-binary#422]
+    @Test
+    public void testEnumSchemaGeneration422() throws Exception
+    {
+        // First, failure due to invalid enum value (when generating as Enum)
+        AvroSchemaGenerator gen = new AvroSchemaGenerator()
+                .enableLogicalTypes();
```

Comment:
@cowtowncoder Enabling logical types is irrelevant for this test. 
I would remove it -  it is misleading as being important, it is not.

Reply:
Makes sense -- I just copied submitted reproduction but agreed it's not pertinent to thing tested. Will remove.